docs/autodoc_submodules.py

```diff
@@ -24,6 +23,12 @@
 exclude_fn_members = {
 }
 
+mod_aditional_doc = {
+    'nvidia.dali.fn.transforms' : "All operators in this module supports only CPU device as they are meant " +
+"to be provided as an input to named operator arguments. Check for more details the relevant " +
```

Comment:
I think we should try to stick with one, consistent naming.
We started using the "argument input" term that we use in the backend, and explain it in the Python layer as named keyword arguments. Please consider using one of those.

Reply:
Done